aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -1,4 +1,9 @@
 module aptos_framework::transaction_context {
+    use std::option::Option;
+
+    /// Transaction context is not available outside of the transaction prologue, execution, or epilogue phases.
+    const E_TRANSACTION_CONTEXT_NOT_AVAILABLE: u64 = 1;
```

Comment:
Do we have a convention to use `ESOME_ERROR` or `E_SOME_ERROR`?

Reply:
@georgemitenkov , `E_SOME_ERROR` or what?

Reply:
Ah, I think it's `ESOME_ERROR`. I will update this.

I accidentally followed this incorrect convention. https://github.com/aptos-labs/aptos-core/blob/37bbc53ee45f7c6d1ad836e612dc81789f86b7d7/aptos-move/framework/aptos-stdlib/sources/cryptography/ed25519.move#L15-L15

Reply:
Done.